### ISSUE TITLE
Better support for .erb comment blocks

### ISSUE THREAD

**Issue by Vuneu** [rubyide/vscode-ruby#190]
### Expected behavior

```erb
<% 
=begin %>
  <p id="notice"><%= notice %></p>
  <h1><%= content %></h1>
<% 
=end %>
```

### Actual behavior
```erb
<%# <p id="notice"><%= notice %></p>
<h1><%= content %></h1> %>
```

**Comment by wingrunr21** [comment]
This is resolved by #260 as the code between the start and end `<% %>` tags will be detected as Ruby (and thus the `=begin` and `=end` block comment syntax will work).

**Comment by wingrunr21** [comment]
This should be fixed now
